src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -49,5 +49,5 @@ case class JesBackendCall(backend: JesBackend,
   def standardParameters = Seq(stderrJesOutput, stdoutJesOutput, rcJesOutput) 
   def rcGcsPath = rcJesOutput.gcs
   def execute: ExecutionResult = backend.execute(this)
-  def downloadRcFile = jesConnection.storage.slurpFile(GoogleCloudStoragePath(callGcsPath, RcFilename))
+  def downloadRcFile = GoogleCloudStoragePath.parse(callGcsPath + "/" + RcFilename).map(jesConnection.storage.slurpFile)
```

Comment:
I'd suggest putting the type ascription on the method now that it's `Try[String]`.